### ISSUE TITLE
CTRL key repetition

### ISSUE THREAD

**Issue by Noomene** [randyrants/sharpkeys#565]
Dear Dev,

First of all, thank you very much for this gem.

My question is regarding the following:

I have remapped the RIGHT CTRL key to work as a DEL key. 
![image](https://github.com/randyrants/sharpkeys/assets/11027939/1c45c5f5-2ea3-46b4-b28d-c571529c9c81)

However, the key **does not repeat** when pressed like a DEL key would.

Is this a built in windows feature? I have windows 11 by the way. I remember I managed to do it somehow on an old windows 7 I had before, but I could not make it work on this new W11 system. Am I missing something?

Thank you so much for your kind reply.

Regards,

**Comment by randyrants** [comment]
I saw on a few sites that Filter keys can sometimes prevent a key from repeating.  Thats under Setting > Accessibility > Keyboard if you wanted to see if it's on but I would think that would prevent Delete from repeating even if you are using it on its own?

**Comment by Noomene** [comment]
Thank you for your reply.
Yes, I did check that. I tried various combinations of filter keys settings without success. It's weird because I did not have this issue on my old system. I'm also sure I only used sharpkeys to fix it, I did not use autohotkey or anything external. Could it W11 is using different scancode for the control key?

**Comment by randyrants** [comment]
No, the scancode has been the same since Windows NT.  They often add new values but almost never change existing values, because of the incompatibility that it would bring across versions.
